lib/toggle/toggle.interface.ts

```diff
@@ -0,0 +1,18 @@
+export interface Props {
+  checked?: boolean;
+  children: React.ReactNode;
+  className?: string;
+  disabled?: boolean;
+  id?: string;
+  modifiers?: string;
+  name?: string;
+  onChange?: () => void;
+  readOnly?: boolean;
+  style?: React.CSSProperties;
+  value: string;
+}
+
+export interface State {
+  checked?: boolean;
+  disabled?: boolean;
```

Comment:
You can remove this line, because disabled is no longer used into state

Reply:
AH! Of course. TY.